testing/pom.xml

```diff
@@ -55,6 +55,11 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
```

Comment:
I am not a fan of this. We are now forcing all users of the testing module to also use assertj. While I am comfortable with using this in our tests, this is expanding our footprint as a library to the users.

Reply:
can this be test scope?

Reply:
removed